hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1032,6 +1033,37 @@ public String getWriteMetaKeyPrefixes() {
     return props.getProperty(WRITE_META_KEY_PREFIXES_PROP);
   }
 
+  /**
+   * Error table configs.
+   */
+  public boolean enableErrorTable() {
+    return Boolean.parseBoolean(props.getProperty(HoodieErrorTableConfig.ERROR_TABLE_ENABLE_PROP));
+  }
+
+  public String getErrorTableBasePath() {
+    return props.getProperty(HoodieErrorTableConfig.ERROR_TABLE_BASE_PATH_PROP);
+  }
+
+  public String getErrorTableName() {
+    return props.getProperty(HoodieErrorTableConfig.ERROR_TABLE_NAME_PROP);
+  }
+
+  public int getErrorTableInsertParallelism() {
```

Comment:
can move this to HoodieErrorTableConfig?

Reply:
I think it is reasonable here, you need to get these values from HoodieWriteConfig